src/layout/List/types.d.ts

```diff
@@ -7,13 +7,14 @@ export interface IPagination {
 }
 
 export interface ILayoutCompList extends ILayoutCompBase<'List'> {
-  tableHeaders?: string[];
+  tableHeaders: { [Col in keyof T]: string };
```

Comment:
I don't think `T` is defined here. It should probably be:

```suggestion
  tableHeaders: Record<string, string>;
```

Reply:
Also, make sure to update the JsonSchema when changing these types! The two definitions must be kept in sync.